### DISCUSSION TITLE
Book Template data does not show in Reading View

### DISCUSSION THREAD

**Discussion by FSCastleberry** [groepl/Obsidian-Templates#7] [Q&A] [answered]
I am using your book template and I can only see the book name and author in the reading view. I can see that the data is available in editing mode. Can point to what may be the issue? I'm running v1.1.16 if that matters. I also verified that I have the required plugins. I'm sure I am missing something simple.

Thanks in advance

**Comment by groepl** [comment]
Are you using the book template together with the Obsidian plugin "Book Search"? See: https://github.com/anpigon/obsidian-book-search-plugin . Also setup the plugin configuration?

Can you please share an example with a screenshot to find the root cause.

**Reply by FSCastleberry** [reply, marked as answer]
The plugin configuration may be the issue. 
![Book search Plugin](https://user-images.githubusercontent.com/128007121/233812890-59610929-3ae7-407e-b119-402ede9c78ca.png)
![Book Template](https://user-images.githubusercontent.com/128007121/233812891-0dd77e86-27c1-464c-a3ac-cfa2e2b06ee6.png)
![Book Template result](https://user-images.githubusercontent.com/128007121/233812893-059c4e4c-6de8-4813-8038-dd6cc48bb04c.png)

**Reply by groepl** [reply]
You wrote: „I can only see the book name and author in the reading view.“ Which version of the template are you using? Here is my template v1.5:

![IMG_0875](https://github.com/groepl/Obsidian-Templates/assets/114583103/1472f8b4-9476-46da-8c62-31dbd54fbab2)

It also uses the input field {{description}}:

# {{Title}}
by [[{{author}}]]

## Summary

{{description}}

**Reply by lexpoig** [reply]
Hello. Great repository of stuff. Interesting. I am trying to create a new Book Note from the template but it does not work. Every time
I try I get an error message saying it couldn't be created and "The properties in the template file could not be read."

I have tried to both insert the template alone in a blank note as well as running the Create New Book Note plugin and THEN trying to insert the Book template. Nothing works. Any thoughts would be appreciated. Thanks!

**Reply by lexpoig** [reply]
Never mind! Figured out I have to configure it in the plugin.
